packages/app/src/hooks/useExplainQuery.tsx

```diff
@@ -0,0 +1,36 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { sendQuery } from '@/clickhouse';
+import {
+  ChartConfigWithDateRange,
+  renderChartConfig,
+} from '@/renderChartConfig';
+
+export function useExplainQuery(
```

Comment:
just noting we probably want to eventually DRY out with https://github.com/hyperdxio/hyperdx/blob/83f9997c03a31c73fd7dfbf03a83d1a34909a483/packages/app/src/BenchmarkPage.tsx, but nothing important rn imo

Reply:
Totally agree. There's probably a reusable component we can abstract out eventually for all queries. There's a pretty solid reusable pattern with the query library we have, but we repeat a lot of the code in various places.